src/components/cc-zone-picker/cc-zone-picker.js

```diff
@@ -35,13 +38,31 @@ export class CcZonePicker extends CcFormControlElement {
   constructor() {
     super();
 
-    /** @type {ZonesSections} array of zones sections */
-    this.zonesSections = [];
+    /** @type {boolean} whether the whole form should be disabled */
+    this.disabled = false;
+
+    /** @type {boolean} whether the form should be in readonly */
+    this.readonly = false;
 
     /** @type {string} current selected zone code */
     this.value = null;
+
+    /** @type {ZonesSections} array of zones sections */
+    this.zonesSections = [];
   }
 
+  /**
+   * @param {CcZonePickerPropertyValues} changedProperties
+   */
+  willUpdate(changedProperties) {
+    if (changedProperties.has('value') && this.value == null) {
+      this.value = this.zonesSections[0].zones[0].code;
```

Comment:
fix: protect the code from weird usage. `this.zoneSections` may be `null` or empty. `this.zoneSections[0].zones` may be `null` or empty. In those cases the `value` should remain `null`.